tests/storage-spec.js

```diff
@@ -155,7 +163,7 @@ describe('languageStorage factory', function () {
                     label: 'Polski',
                 },
                 {
-                    id: 'en_us',
+                    id: 'en',
```

Comment:
I think the culture should still be matched if it's there. This non-culture match feature should be additional if with-culture isn't found in the choices.